plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -213,46 +216,27 @@ public void testColumnComment()
     @Test
     public void testPredicatePushdown()
     {
+        // varchar inequality
+        assertThat(query("SELECT regionkey, nationkey, name FROM nation WHERE name != 'ROMANIA' AND name != 'ALGERIA'"))
+                .isFullyPushedDown();
+
         // varchar equality
         assertThat(query("SELECT regionkey, nationkey, name FROM nation WHERE name = 'ROMANIA'"))
                 .matches("VALUES (BIGINT '3', BIGINT '19', CAST('ROMANIA' AS varchar(25)))")
-                // SQL Server is case insensitive by default
-                .isNotFullyPushedDown(FilterNode.class);
+                .isFullyPushedDown();
 
         // varchar range
         assertThat(query("SELECT regionkey, nationkey, name FROM nation WHERE name BETWEEN 'POLAND' AND 'RPA'"))
                 .matches("VALUES (BIGINT '3', BIGINT '19', CAST('ROMANIA' AS varchar(25)))")
-                // SQL Server is case insensitive by default
+                // We are not supporting range predicate pushdown for varchars
                 .isNotFullyPushedDown(FilterNode.class);
 
         // varchar IN without domain compaction
         assertThat(query("SELECT regionkey, nationkey, name FROM nation WHERE name IN ('POLAND', 'ROMANIA', 'VIETNAM')"))
```

Comment:
Can we have additional test with `NOT IN`

Reply:
Test case added